tests/Bridge/Symfony/Bundle/DataCollector/RequestDataCollectorTest.php

```diff
@@ -181,7 +181,12 @@ public function testWithResourceWithTraceables()
 
         $dataProvider = $dataCollector->getCollectionDataProviders();
         foreach ($dataProvider['responses'] as $class => $response) {
-            $this->assertStringStartsWith('class@anonymous', $class);
+            if (\PHP_VERSION_ID >= 80000) {
+                $this->assertStringStartsWith('ApiPlatform\\Core\\DataProvider\\CollectionDataProviderInterface@anonymous', $class);
```

Comment:
Maybe can we use the `::class` constant? Or just use `assertStringContainsString()` to avoid the condition totally.